scripts/subquery-e2e.py

```diff
@@ -121,3 +120,33 @@
 """
 Legacy Bridge Swap
 """
+url = "https://github.com/fetchai/fetch-ethereum-bridge-v1/releases/download/v0.2.0/bridge.wasm"
+contract_request = requests.get(url)
```

Comment:
Would it make sense to only make the request if it isn't found on the filesystem?

Reply:
You're right, thanks!